### DISCUSSION TITLE
一个建议

### DISCUSSION THREAD

**Discussion by xingnengmao666** [Meloong-Git/PCL#1351] [提问] [answered]
移动网络在登陆正版时会出现白屏的现象，建议添加一个“自动将DNS解析服务器改为4.2.2.2和8.8.8.8”的选项，不用每次去百度上翻教程，这样能方便各位萌新更好的去登陆账号享受游戏的乐趣

**Comment by 212112111zwl** [comment, marked as answer]
请在issues中提交建议，并使用一个更具体地issue标题，谢谢~

**Comment by WForst-Breeze** [comment]
如有新功能建议，请提交[新功能提案](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+%E6%96%B0%E5%8A%9F%E8%83%BD%2C%E6%96%B0%E5%8F%8D%E9%A6%88&template=feature.yml)，感谢您的配合~
